nucypher/blockchain/eth/agents.py

```diff
@@ -44,6 +47,13 @@ def clear(mcs):
     def agents(cls):
         return cls.__agents
 
+    @classmethod
+    def get_agent(mcs, cls):
+        try:
```

Comment:
what is an "mcs"?

Reply:
it's a conventional first-arg name for a meta-classmethod.